pkg/tfbridge/diff.go

```diff
@@ -386,3 +386,47 @@ func makeDetailedDiffExtra(
 		collectionDiffs: collectionDiffs,
 	}
 }
+
+func reportRawTFDiff(ctx context.Context, attrs map[string]shim.ResourceAttrDiff) {
```

Comment:
The not great part about using attrs map[string]shim.ResourceAttrDiff is that I can think of at least one false positive case (oof,I'll need to dig up that one). Where the InstanceDiff object returns change details but TF actually has a no-change plan which is the expected plan.